tests/e2e/accounts/accounts_users/test_sync_accounts_users.py

```
@@ -0,0 +1,191 @@
+        create_user_group_data
+    )
+    yield created_account_user_group
+
+    try:
+        users = mpt_vendor.accounts.accounts.users(account_id=account_id)
+        users.groups(user_id=created_account_user_data.id).delete(user_group_data.id)
+    except MPTAPIError as error:
+        print(f"TEARDOWN - Unable to delete account user group: {error.title}")  # noqa: WPS421
+
+
+def test_get_account_user_by_id(mpt_vendor, user_id, account_id):  # noqa: AAA01
```

Ruff (0.14.6)

Unused `noqa` directive (unknown: `AAA01`)

Remove unused `noqa` directive

(RUF100)